proto/heartbeat.go

```diff
@@ -23,6 +23,13 @@ import (
 	"time"
 )
 
+// TODO(marc): we should assert
+// var _ security.AnyUserRequest = &PingRequest{}
+// here, but we need to break cycles first.
+
+// AnyUser implements security.AnyUserRequest.
+func (*PingRequest) AnyUser() {}
```

Comment:
This `PingRequest` does too much (HLC updates, etc). If we want to expose a user-level ping operation we should have a new RPC, probably defined at the SQL layer. But we can probably just use an empty `sql/driver.Request` instead.